{
    "config": {
        "abort": {
            "already_configured": "El dispositivo ya est\u00e1 configurado",
            "reauth_successful": "La reautenticaci\u00f3n fue exitosa"
        },
        "error": {
            "cannot_connect": "Error al conectar",
            "invalid_auth": "Autenticaci\u00f3n no v\u00e1lida",
            "unknown": "Error inesperado"
        },
        "step": {
            "user": {
                "data": {
                    "api_key": "Clave API"
                }
            }
        }
    }
}